
Tweak the default config to get installbuilddir right.
(should really make the RedHat layout DTRT again and
use that layout instead)

--- httpd-2.1.10/config.layout.layout
+++ httpd-2.1.10/config.layout
@@ -20,7 +20,7 @@
     mandir:        ${prefix}/man
     sysconfdir:    ${prefix}/conf
     datadir:       ${prefix}
-    installbuilddir: ${datadir}/build
+    installbuilddir: ${libdir}/httpd/build
     errordir:      ${datadir}/error
     iconsdir:      ${datadir}/icons
     htdocsdir:     ${datadir}/htdocs
